example/traffic_mgmt/odp_traffic_mgmt.c

```diff
@@ -753,6 +755,8 @@ static int destroy_tm_queues(void)
 	int class;
 	int ret;
 
+	odp_tm_stop(odp_tm_test);
```

Comment:
I think it would be better to raise odp_tm_start() and odp_tm_stop() calls to the top level so that they stand out better. The stop is anyway not really logically part of queue destroying so it seems a bit out of place here.